### DISCUSSION TITLE
How do I update my Miyoo Mini + to install Onion?

### DISCUSSION THREAD

**Discussion by Q113P** [OnionUI/Onion#888] [Q&A]
my stock firmware on my Miyoo is 202305052130 and I'm having trouble updating it so I can install Onion. I got the Miyoo mini upgrade 20220419, took a new sandisk sd card formatted it to fat32, than put the firmware from the firmware folder and also the stuff from TF card 1. However it didn't work so I went and searched for a newer  most recent tutorial video. I found one from Retro Handheld. I added their updated img for miyoo mini + on the sd card. Then I finally got a firmware update screen. However about 5 minutes the screen glitched then it glitched again in another 5 minuets. I never got a charging screen so I unplugged the device like 5 minutes ago so it doesn't overheat and cause a fire (since it was more than halfway full anyway). Now my device doesn't boot at all and only a black screen. I need help not only to get it working again but to install Onion on it too. Can I pleasev get help.

**Comment by Aemiii91** [comment]
Fw version `202305052130` is the latest version for Mini+. You shouldn't be trying to downgrade, especially not to 20220419 which is for regular Mini only.

**Comment by Q113P** [comment]
oh it was? dang I had a feeling. Is there a way to fix it? like try to find that fw version and reinstall it.

**Reply by Aemiii91** [reply]
The only one that's available for download is `202303262339`, afaik.

**Reply by Q113P** [reply]
oh dang. I'm going to see if Onion could still install. 
edit:
dang can't install it either. I hope there's somthing else i can do

**Comment by Q113P** [comment]
I no longer need this since I figured it out and reinstalled both firmwares and have the device working now

**Reply by legendsiew** [reply]
Hi, i have the same firmware as urs,  maybe whether i should downgrade to 202303262339 or i can install onion straight away without changing the latest firmware i have
